src/Pages/LoginPage/LoginPage.tsx

```diff
@@ -1,14 +1,23 @@
 import './LoginPage.scss';
+import { Link } from 'react-router-dom';
+import { loginSlice } from './redux/loginPageSlice';
+import { useDispatch } from 'react-redux';
 
 export const LoginPage = () => {
+  const dispatch = useDispatch();
+
+  const onLoginPress = (e: any) => {
```

Comment:
you must not use "any" =)
if you don't use event's object, you can leave these brackets empty

Reply:
fixed